janus_core/calculations/single_point.py

```diff
@@ -386,11 +306,15 @@ def run(
             properties = [properties]
 
         for prop in properties:
-            if prop not in ["energy", "forces", "stress"]:
+            if prop not in get_args(Properties):
                 raise NotImplementedError(
                     f"Property '{prop}' cannot currently be calculated."
                 )
 
+        # If none specified, get all valid properties
+        if len(properties) == 0:
```

Comment:
Could this be `not properties`?

Reply:
Yes, probably. This was mostly a like for like replacement for `if "energy" in properties or len(properties) == 0:` etc.

I've added a similar change in #225, where I'm doing more refactoring of SinglePoint, but happy to do it here if it makes more sense.